wallet/src/Pos/Aeson/ClientTypes.hs

```diff
@@ -73,3 +74,15 @@ instance ToJSON ClientInfo where
             , "cabalVersion" .= showVersion ciCabalVersion
             , "apiVersion" .= ciApiVersion
             ]
+
+instance FromJSON NewBatchPayment where
+    parseJSON = withObject "NewBatchPayment" $ \o -> do
+        npbFrom <- o .: "from"
+        npbTo <- fmap toList $ withArray "NewBatchPayment.to" collectRecipientTuples =<< o .: "to"
+        npbInputSelectionPolicy <- o .:? "inputSelectionPolicy" .!= def
```

Comment:
Also this field is already called in another way
https://github.com/input-output-hk/cardano-sl/pull/1919/files#diff-250ee2a3dd2e1fe48fb350c1fcd92947R45

But I'm okay with new name if you have some reason for it in mind

Reply:
Will change the name. Added to todo

Reply:
Fixed